_data/accessibility-tests/banner.yml

```diff
@@ -0,0 +1,112 @@
+      You can use the `enter` and `spacebar` keys to open and close the banner.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: Banner does not trap keyboard focus.
+    summary_additional: |
+      When you navigate with a keyboard, you can easily move into and out of the banner,
+      regardless of whether the banner is open or closed.
+      When you navigate out of an open banner using your keyboard, it doesn't automatically close.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.2
+  - summary: Banner is the first interactive object after the main navigation.
+    summary_additional: |
+      When you navigate your site with a keyboard and skip navigation,
+      the "Here's how you know" banner button is the first focusable item on the page.
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.3
+  - summary: Focus indicator is clearly visible on banner.
+    summary_additional: |
+      When you use a keyboard to tab into the banner,
+      the "Here's how you know" button has a visible outline or other clear indication of focus.
+      On a mobile device, this focus indicator should surround the entire banner element.
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader announces button.
+    summary_additional: |
+      When you use a screen reader and access the banner,
+      the screen reader uses the word "button" to announce the "Here's how you know" button.
+    test_status:
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: Screen reader announces collapsed state.
+    summary_additional: |
+       When you first access the "Here's how you know" button while using a screen reader,
+       the screen reader announces that it is "collapsed."
+       When activated, the screen reader announces "expanded".
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 4.1.2
+  - summary: Screen reader does not announce decorative icons.
+    summary_additional: |
+       When you access and open the banner while using a screen reader,
+       you don't hear any information about the flag, blue government building, or green lock icon.
+       When you view the alt text for the flag, blue government building, and green icon lock,
+       you see that they have been marked as decorative.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.1.1
+  - summary: Screen reader announces lock icon.
+    summary_additional: |
+       When you open the banner with a screen reader,
+       the screen reader uses the words "lock icon" to announce the lock icon.
```

Comment:
Realizing that I think we should specify "white lock icon inside the parentheses" at the end of this sentence--otherwise people might confuse it with the other lock icon. Or they might overlook it altogether because it's not as obvious as the other lock icon. I think that without more specificity this test might cause confusion. (@amyleadem)

Reply:
I updated the copy to say "lock icon inside the parentheses" in [fd7120b](https://github.com/uswds/uswds-site/pull/2687/commits/fd7120b411d3f6a5e908bdcf3de1279fdbe50a74). I did not use the word "white" because the [default implementation](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/?path=/story/components-banner--default) of the component has a black icon. Our site uses a different color combination than the standard.